CRM/Upgrade/Incremental/php/FiveFortyThree.php

```diff
@@ -111,7 +111,15 @@ public function upgrade_5_43_alpha1(string $rev): void {
       'updateMessageToken', '', 'contribution.contribution_status', 'contribution.contribution_status_id:label', $rev
     );
     $this->addTask('Update campaign token in saved message templates',
-      'updateMessageToken', '', 'contribution.campaign', 'contribution.campaign_id:label', $rev
+      'updateMessageToken', '', 'contribution.campaign', 'contribution.campaign_id:label', $rev);
+    $this->addTask('Update start date token in event badges',
+      'updatePrintLabelToken', 'event.start_date', 'event.start_date|crmDate:"%B %E%f', $rev
```

Comment:
Here's the typo (note the missing quotation mark after `%f`) that I'm referencing in my note on core#3952.